cupy_setup_build.py

```diff
@@ -637,6 +637,13 @@ def get_ext_modules(use_cython=False):
 def _nvcc_gencode_options(cuda_version):
     """Returns NVCC GPU code generation options."""
 
+    if sys.argv == ['setup.py', 'develop']:
+        return []
+
+    envcfg = os.getenv('CUPY_NVCC_GENERATE_CODE', None)
+    if envcfg is not None:
```

Comment:
In ChainerX, the build configuration skips adding the argument if `envcfg` is empty string. I have no idea which behavior is preferred (`""` will override the default behavior and add just a `--generate-code=`). Any idea?

Reply:
Thanks for mentioning.
I think empty string should be ignore too will fix this.